frontend/kesaseteli/employer/src/components/application/steps/step1/Step1Employer.tsx

```diff
@@ -0,0 +1,54 @@
+import CompanyInfo from 'kesaseteli/employer/components/application/companyInfo/CompanyInfo';
+import ActionButtons from 'kesaseteli/employer/components/application/form/ActionButtons';
+import TextInput from 'kesaseteli/employer/components/application/form/TextInput';
+import StepForm from 'kesaseteli/employer/components/application/StepForm';
+import useApplicationApi from 'kesaseteli/employer/hooks/application/useApplicationApi';
+import { useTranslation } from 'next-i18next';
+import React from 'react';
+import FormSection from 'shared/components/forms/section/FormSection';
+import EmployerApplication from 'shared/types/employer-application';
+
+import { $EmployerBasicInfo } from './Step1Employer.sc';
+
+const Step1Employer: React.FC = () => {
+  const { t } = useTranslation();
+  const { updateApplication } = useApplicationApi();
+
+  const onSubmit = (draftApplication: EmployerApplication): void => {
+    updateApplication(draftApplication);
+  };
+  const stepTitle = t('common:application.step1.header');
+  return (
+    <StepForm stepTitle={stepTitle}>
+      <FormSection
+        header={stepTitle}
+        tooltip={t('common:application.step1.tooltip')}
+      >
+        <CompanyInfo />
+      </FormSection>
+      <FormSection>
+        <$EmployerBasicInfo>
+          <TextInput
+            id="invoicer_name"
+            validation={{ required: true, maxLength: 256 }}
+          />
+          <TextInput
+            id="invoicer_email"
+            validation={{
+              required: true,
+              maxLength: 254, // eslint-disable-next-line security/detect-unsafe-regex
+              pattern: /^(([^\s"(),.:;<>@[\\\]]+(\.[^\s"(),.:;<>@[\\\]]+)*)|(".+"))@((\[(?:\d{1,3}\.){3}\d{1,3}])|(([\dA-Za-z-]+\.)+[A-Za-z]{2,}))$/,
```

Comment:
`Yup` has a built-in method for validating emails.

Reply:
Including yup seem to add a lot of boilerplate: https://react-hook-form.com/advanced-usage#CustomHookwithResolver compared to react-hook-form builtin validation.
But these validation objects could be moved to own file. But I rather do it when meaking step2 if I have to start copy-pasting stuff.